src/app/EventManagement.h

```diff
@@ -419,6 +423,16 @@ class EventManagement
      */
     CHIP_ERROR EnsureSpaceInCircularBuffer(size_t aRequiredSpace);
 
+    /**
+     * @brief Iterate the event elements inside event tlv and mark the fabric index as 0 if matching with obsolete fabric index
+     * converted from apContext
+     *
+     * @param[in] aReader  event tlv reader
+     * @param[in] apContext  the obsolete fabric index
```

Comment:
Please document that this is a `FabricIndex *`

Reply:
done, thanks